tests/adapters/shell/terminal_client_test.py

```diff
@@ -198,6 +226,32 @@ class TelnetClientTest(TerminalClientTest):
     client = TelnetClient
     port = 10011
 
+    @patch('netman.adapters.shell.telnet._connect')
+    @patch('netman.adapters.shell.telnet.TelnetClient._login', Mock())
+    def test_changing_default_connect_timeout(self, connect_method_mock):
+        TelnetClient(**self._get_some_credentials())
+        self.assertEqual(60, connect_method_mock.call_args[0][2])
```

Comment:
You could use assert_called_with instead (same applied to all other uses of this):

connect_method_mock.assert_caled_with(60)
or
connect_method_mock.assert_caled_once_with(60)